tfx/extensions/google_cloud_ai_platform/runner.py

```diff
@@ -156,6 +156,7 @@ def start_cloud_training(input_dict: Dict[str, List[types.Artifact]],
                          exec_properties: Dict[str, Any],
                          executor_class_path: str,
                          job_args: Dict[str, Any],
+                         job_labels: Dict[str, Any],
```

Comment:
can we make job_lables optional (same elsewhere), and comment it's for AI platform only? Thanks!